### DISCUSSION TITLE
不懂就问，

### DISCUSSION THREAD

**Discussion by yzyz5840** [MetaCubeX/mihomo#1287] [Q&A]
gz文件咋弄成apk文件来适应安卓手机安装哦

**Comment by xiandu999-yu** [comment]
面具模块解君愁
![3087d273a78ccaff4bb1e9972e2ba2a7583c9f11 png@48w_48h](https://github.com/MetaCubeX/mihomo/assets/58256481/83ff8795-9e1e-46aa-a665-983f63727dcb)

**Reply by yzyz5840** [reply]
虽然不懂还是要谢谢回复
